Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -1785,6 +1809,15 @@ void EditorActionsHandler::OnMenuBindingHook()
         }
     }
 
+    // Tools
+    {
+        m_menuManagerInterface->AddActionToMenu(
+            EditorIdentifiers::ToolsMenuIdentifier,
+            "o3de.action.tools.luaEditor",
```

Comment:
question: Could we use the constant `actionIdentifier` here from above instead of duplicating from the above?

Reply:
Oh, it's a different function. So we could if I moved it outside the function's scope...
Not worth holding this PR for since it passed AR, but I'll think about it if it makes sense for a future refactoring of this class.